organizational/conduct.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Hacker Society [#hacsoc]_ is, as Tim wrote in :doc:`hacsoc`, about an open
+society:
+
+  An open society promotes freedom of thought and expression, personal
+  responsibility, individualism, cooperation, and equality of all citizens.
+
+As such, we expect that our members behave in a way that allows us to achieve
+this.  Participants in HacSoc events are expected to treat each other with
+dignity and respect for each other's perspectives, experiences, and knowledge.
+In particular, participants should recognize that people of all skill levels are
+welcome, and should always be willing to help others learn.
+
+We want Hacker Society to be a place where people can be comfortable regardless
+of gender, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, race, age, or religion.  As such, we do not
+tolerate harassment.  This includes, but is not limited to:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goal of providing an open atmosphere for discussion and debate
+has some tension with the goal of providing a welcoming environment.  We expect
+that debate and constructive criticism are part of this open atmosphere.
+However, when these discussions devolve into attacks against a person and/or
+their intrinsic characteristics, we believe that this is unacceptable.
```

Comment:
change to:

Necessarily, the goals of learning, community, and equality are in tension
with the goals individual empowerment, freedom of thought, and freedom of
expression. We strive for politeness and civil discourse but we also strive to
make *all* welcome. Even those with unpopular ideas or thoughts.  However, in
order to foster our community certain limits on free expression must be
observed. We expect that debate and constructive criticism are part of this
open atmosphere. We expect that some members of the community may express
ideas which are not politically correct. We expect that some ideas expressed
by a minority may be deeply unpopular and distastful to the majority.
However, if these ideas or discussions devolve into attacks against a person
and/or their intrinsic characteristics, we believe that this is unacceptable.

Here is an example of an idea which attacks a non-intrinsic characteristic: I
hate how Pat is so bad at using email he always replies before the quoted
text instead of after. The way Pat uses email is not intrinsic to Pat an may
be violating other established community norms. However, saying: I hate how
Pat always wears yellow shirts so ugly! That is saying something about an
intrinsic to Pat their preference for yellow shirts. The former is allowed the
latter would be viewed as unwelcome. In the same way saying: I love how Pat
always uses vim, is fine but saying: I love how Pat always wears green it is
so hot, is not fine.